cluster/swarm/cluster.go

```diff
@@ -968,6 +968,31 @@ func (c *Cluster) BuildImage(buildContext io.Reader, buildImage *types.ImageBuil
 	return nil
 }
 
+// RefreshEngines refreshes all containers in the cluster (UCP-specific)
```

Comment:
I see your point, and can't think of a better way to do this given the Docker client restrictions. We should make sure to document this properly though.